s3_file_field/_multipart_minio.py

```diff
@@ -12,10 +14,19 @@ def __init__(self, storage: MinioStorage):
         # for pre-signing URLs when it exists
         self._signing_client: minio.Minio = getattr(storage, 'base_url_client', storage.client)
 
-    def _create_upload_id(self, object_key: str) -> str:
+    def _create_upload_id(
+        self,
+        object_key: str,
+        content_type: Union[str, None] = None,
+    ) -> str:
+        metadata = {}
```

Comment:
Since we know the file name at this point, can we set `Content-Disposition` too?